integrations-and-supported-tools/azure-ml/build_pipeline.py

```diff
@@ -0,0 +1,140 @@
+import hashlib
+import os
+import time
+
+from azure.ai.ml import Input, MLClient, Output, command, dsl
+from azure.ai.ml.constants import AssetTypes
+from azure.ai.ml.entities import Data
+from azure.identity import DefaultAzureCredential, InteractiveBrowserCredential
+
+DATA_PREP_SRC_DIR = "components/data_prep"
+TRAIN_SRC_DIR = "components/train"
+VALID_SRC_DIR = "components/validate"
+
+NEPTUNE_PROJECT = "common/project-time-series-forecasting"  # change to your own Neptune project
+NEPTUNE_CUSTOM_RUN_ID = hashlib.md5(str(time.time()).encode()).hexdigest()
+neptune_api_token = os.environ["NEPTUNE_API_TOKEN"]
+
+AZURE_SUBSCRIPTION_ID = "<YOUR SUBSCRIPTION ID>"
+AZUREML_RESOURCE_GROUP_NAME = "<YOUR RESOURCE GROUP NAME>"
+AZUREML_WORKSPACE_NAME = "<YOUR WORKSPACE NAME>"
+
+
+def compose_pipeline(
+    compute_target="cpu-cluster",
+    custom_env_name="neptune-example",
+    custom_env_version="2",
+    neptune_project="common/project-time-series-forecasting",
```

Comment:
Is this supposed to be hardcoded, or could it use `NEPTUNE_PROJECT`?

Reply:
Should use `NEPTUNE_PROJECT`